homeassistant/components/camera/webrtc.py

```diff
@@ -156,6 +156,15 @@ def async_close_session(self, session_id: str) -> None:
         """Close the session."""
         return  ## This is an optional method so we need a default here.
 
+    async def async_get_image(
```

Comment:
We need a dev docs PR for this method, right?

Reply:
Other methods are also not document in the dev docs and therefore I would merge it without any docs